assets/src/edit-story/components/library/panes/media/mediaElement.js

```diff
@@ -260,14 +260,20 @@ const MediaElement = ({
 
   const { lengthFormatted, poster, mimeType } = resource;
   return (
-    <Container onPointerEnter={onPointerEnter} onPointerLeave={onPointerLeave}>
+    <Container
+      data-testid="mediaElement"
```

Comment:
Do we still need the `testid`?

Reply:
At least one old e2e test is using it. Should be removed when we fully transition to getByRole/Label.